### DISCUSSION TITLE
fetch failed

### DISCUSSION THREAD

**Discussion by ChenTurboo** [danny-avila/LibreChat#864] [Q&A]
Running Bing and Sydney may not answer, reply：Oops! Something went wrong. Please try again in a few moments. Here's the specific error message we encountered: fetch failed

**Comment by danny-avila** [comment]
This is not reproducible on my end, as is the case with many Bing issues. However, there are certainly many things you can try.

You can search previous issues for this, as well as look for common solutions posted on our discord. To join the discord server, use this link: https://discord.librechat.ai/

Once you join, @fuegovic has organized these into a discord thread which you can visit here: https://discord.com/channels/1086345563026489514/1143941308684177429

You can also search the issues on discord.

It would help to note what you have tried once visiting possible solutions. I'll implement SOCKS5 proxy support which should help assuming you have an IP outside of the US as well.
